com.github.git-cola.yml

```diff
@@ -0,0 +1,51 @@
+app-id: com.github.git-cola
+runtime: org.kde.Platform
+runtime-version: '5.15'
```

Comment:
```suggestion
runtime-version: '5.15-21.08'
```


5.15 is EOL

Reply:
-21.08 is missing Python.

Reply:
not it's not. It's python 3.9

5.15 is EOL anyway.

Reply:
I misinterpreted the error. It was missing LLVM, but there is an extension for that.